app/notifications/new_assignment_notification.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class NewAssignmentNotification < Noticed::Base
+  deliver_by :database, association: :noticed_notifications
+
+  def message
+    # user = params[:group_member.user]
+    #assignment = params[:assignment]
+    assignment = Assignment.all
```

Comment:
why `Assignment.all`?
assignment is in params, right?
you will get it `assignment = params[:assignment]`

Reply:
![image](https://user-images.githubusercontent.com/85568177/196000605-6c8a4c7c-5681-479f-bfb4-e5111e48a548.png)
I get this error on every page when I use
 assignment = params[:assignment]

Reply:
It seems you have deleted the assignment which was saved in `noticed_notifications`!
The simple thing you can do, (as you are working on development)
in terminal:
- `rails c`
- `NoticedNotification.destroy_all`

And try again this!

Also you can ping me on slack for these issues!

Reply:
Thanking you for helping me out so much @vedant-jain03 . This fix worked... (although I had to delete all assignments too)
The redirects are working.
![Screenshot 2022-10-16 at 1 04 14 PM](https://user-images.githubusercontent.com/85568177/196023900-2bf98842-ce05-4f64-9f19-c3026cb37534.png)
And notifications are also showing for new assignments
![Screenshot 2022-10-16 at 1 19 15 PM](https://user-images.githubusercontent.com/85568177/196024400-59626c2d-35d3-441b-98da-f2bbb08381c2.png)

However ... the green dot was overlapping with notification as mentioned in #3313 ... I came up with a quick fix for now for this
![Screenshot 2022-10-16 at 1 21 01 PM](https://user-images.githubusercontent.com/85568177/196024463-527f3c43-d286-4603-9099-f575b4d59e90.png)